aten/src/ATen/native/Activation.cpp

```diff
@@ -5,6 +5,7 @@
 #include <ATen/Dispatch.h>
 #include <ATen/NativeFunctions.h>
 #include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
```

Comment:
nit: is this header needed, now that you have moved the loops to the `cpu/` folder?